src/Config/CliBuilder.ts

```diff
@@ -85,7 +86,20 @@ export class CliBuilder {
       );
 
     return commands
-      .reduce((acc: Argv, item: CommandModule) => acc.command(item), cli)
+      .reduce((acc: Argv, item: CommandModule) => {
+        const handler = item.handler.bind(item);
+
+        item.handler = (args) =>
+          runWithAsyncContext(() => {
+            setContext('args', args);
```

Comment:
Add to the context and/or tags information about the running command

Reply:
@derevnjuk it's already included in the args

<img width="660" alt="image" src="https://github.com/NeuraLegion/bright-cli/assets/9965996/a159fee9-ec74-4351-b66e-b083f56d1795">

Reply:
I see 💪🏾